onnxruntime/contrib_ops/rocm/bert/fast_gelu_impl.cu

```diff
@@ -82,6 +82,84 @@ __global__ void FastGeluKernel2(const half2 a, const half2 b, const half2 c, int
   }
 }
 
+template <unsigned TPB>
+__global__ void FastGeluKernel4Bias(const half2 a, const half2 b, const half2 c, int input_length, int bias_length, 
+		                    const half* input, const half* bias, half* output) {
+  const int idx = blockIdx.x * TPB + threadIdx.x;
+
+  const float2* input_cast = reinterpret_cast<const float2*>(input);
+  const float2* bias_cast = reinterpret_cast<const float2*>(bias);
+  float2* output_cast = reinterpret_cast<float2*>(output);
+
+  const half2 two2 = __floats2half2_rn(two, two);
```

Comment:
Curious: Why not just `[__float2half2_rn](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH____HALF__MISC.html#group__CUDA__MATH____HALF__MISC_1ge40813c17ab4b0779764e2e5e3014019)` since both halves are going to be populated with the same value ?

Reply:
Yes, thanks for pointing this out.